### ISSUE TITLE
`@lends` doesn't work when the `new` operator apears before an expression

### ISSUE THREAD

**Issue by kerimdzhanov** [jsdoc3/jsdoc#1130]
The `@lends` tag won't mark object if `new` operator appears, here is an example:

``` js
'use strict';

var mongoose = require('mongoose'),
    Schema = mongoose.schema,
    ObjectId = Schema.ObjectId;

/**
 * @class User
 */
var UserSchema = new Schema(/** @lends User# */{
  /** @type String */
  email: String,

  /** @type String */
  passwordHash: String,

  /** @type String */
  passwordSalt: String,

  /** @type Role[] */
  roles: [{ type: ObjectId, ref: 'Role' }]
});
```

And if we omit the `new` operator (fortunately some libs supports factory-like constructors, as well as mongoose), the `@lends` works as expected.

**Comment by hegemonic** [comment]
Very strange. Thanks for reporting this issue--this is definitely not the intended behavior. I'll investigate.

**Comment by kerimdzhanov** [comment]
Thank you!

**Comment by purtuga** [comment]
Just switched over a project to use jsdoc3 and am experiencing a similar issue... none of the class members are being output when the `@lends` is used. My case is slightly different from what @kerimdzhanov reported, but I think it may be related - thus why I did not open a new issue.

Here is an example:

```javascript
/**
 * A sliding panel widget.
 *
 * @class SlidingPanel
 *
 * @param {Object} options
 * @param {DOMElement} options.appendTo
 * @param {DOMElement} [options.content]
 * @param {String} [options.position="left"]
 *  Possible values: `left` (default), `right`.
 *
 */
SlidingPanel = /**@lends SlidingPanel.prototype*/{
    init: function(options){},
    
    /**
     * make panel visible
     */
    show: function(){}
}
```

Thanks.

/paul

**Comment by purtuga** [comment]
Looks like there is a pending contribution to fix this.  Hope it gets accepted soon.

**Comment by jscharett** [comment]
+1

**Comment by hegemonic** [comment]
This issue is fixed on master—we now use the [Babylon](https://github.com/babel/babylon) parser and its built-in comment-attachment feature, rather than using custom code for attaching comments. The fix will be included in JSDoc 3.5.0.
